prep-javascript-dom-creation/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript DOM Creation</title>
+</head>
+
+<body>
+  <div id="container">
+
+  </div>
+  <script>
+    // Your code here
+var $container = document.getElementsById('container');
+var $heading = document.createElement('h1');
+    $heading = ('Hello World!');
```

Comment:
You want to assign the string to the `textContent` property of the `$heading` object.

```js
$heading.textContent = 'Hello World!';
```